site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -17,7 +17,7 @@ layout: ../../layouts/ComponentLayout.astro
 - [HIDs and Interest](#hids-and-interest)
   - [Mouse](#mouse)
   - [Keyboard](#keyboard)
-    - [The `keyboard-focusable` CSS property](#the-keyboard-focusable-css-property)
+    - [`interactivity: no-keyboard`](#interactivity-no-keyboard)
```

Comment:
Not going to block on bikeshedding but I think it'd be good to name this in terms of focusability rather than keyboard?

For example, no-keyboard could mean no keydown or keyup events fire on it? No escape key etc.

Reply:
That's fair. Perhaps `not-focusable` ?

Reply:
...but the problem is that "keyboard" should be in there somewhere, because it's really only about **keyboard** focusability. So perhaps `not-keyboard-focusable`. Long, but descriptive.